### DISCUSSION TITLE
Quick clarification (maybe stupid question)

### DISCUSSION THREAD

**Discussion by Corsair-63** [peterantypas/maiana#30] [Q&A]
I still have not a clear picture of the system, but derived from yours pictures, is the system designed as below:
from ADE (Above deck equipment) to BDE (Below Deck Equipment):

ADE
- VHF aerial 
- extender of aerial with Transceiver PCB inside
ADE>BDE
- RJ-45 cable
BDE
- Control box with power supply input.
- USB/COM port connection to PC either multiplexor.

if transceiver is inside the aerial extender I guess that GNSS won't need aerial itself, because it will receive enough signal where it would be locate?

do I get the right picture of the whole system? or is it planned another layout?

sorry to bother you, just for making my mind clear ;-)

**Comment by peterantypas** [comment]
That's the architecture, yes.
